internal/kafka/command_link_describe.go

```diff
@@ -109,11 +109,9 @@ func printHumanTaskOuts(cmd *cobra.Command, taskOuts []serializedTaskOut) error
 		return nil
 	}
 	list := output.NewList(cmd)
-	for i := range taskOuts {
-		t := taskOuts[i]
+	for _, t := range taskOuts {
 		errs := make([]string, 0)
-		for i := range t.Errors {
-			eo := t.Errors[i]
+		for _, eo := range t.Errors {
```

Comment:
What does "eo" mean? Can we just call this error?

Reply:
Yeah done.